static/js/components/AutoComplete.js

```diff
@@ -473,6 +473,7 @@ class AutoComplete extends Component {
         onMouseDown={this.handleMouseDown}
         style={Object.assign(styles.menu, menuStyle)}
         listStyle={Object.assign(styles.list, listStyle)}
+        className="autocomplete-options"
```

Comment:
oh no it is not, I had a helper method earlier that I was using this class for to pick out all the `<li>` elements comprising the autocomplete list, but it turned out to be easier to just fill in the text fields.